exid-types.json

```diff
@@ -105,4 +105,12 @@
     "change-controller": "wikitree.com",
     "reference": "https://www.wikitree.com/wiki/Help:WikiTree_ID"
   },
+  {
+    "label": "GOV ID",
+    "type": "http://gov.genealogy.net/",
+    "description": "Historical Gazeteer place ID",
+    "contact": "gov-support@genealogy.net",
+    "change-controller": "CompGen",
+    "reference": "https://gedcom.io/exid-type/GOV-ID"
```

Comment:
I get 404 not found at this URL

Reply:
How about using https://wiki.genealogy.net/GOV
(which is what "About the project" links to from http://gov.genealogy.net/search/name)

Reply:
the proposed YAML file links to https://wiki.genealogy.net/GOV. However the YAML file ist not yet uploaded, so the 404 occurs.